.github/workflows/reusable-e2e.yml

```diff
@@ -88,7 +88,7 @@ jobs:
           sleep 60
           echo "Running Playwright Tests..."
           mkdir -p playwright/.auth
-          OKTA_USER_NAME=${{secrets.OKTA_USER_NAME}} OKTA_PASSWORD=${{secrets.OKTA_PASSWORD}} TARGET_HOST=https://${{ inputs.webappName }}.azurewebsites${{ vars.AZ_HOSTNAME_SUFFIX }} CAMS_LOGIN_PROVIDER=${{ vars.CAMS_LOGIN_PROVIDER }} npx playwright test --reporter=list,html
+          OKTA_USER_NAME=${{secrets.OKTA_USER_NAME}} OKTA_PASSWORD=${{secrets.OKTA_PASSWORD}} TARGET_HOST=https://${{ inputs.webappName }}.azurewebsites${{ vars.AZ_HOSTNAME_SUFFIX }} CAMS_LOGIN_PROVIDER=mock npx playwright test --reporter=list,html
```

Comment:
We need a TODO: Figure out how we can utilize our OktaTest user for user role mapping

Reply:
Also, I just realized in order for these to pass we would need to also deploy with mock. @btposey Is the intent to just deploy with mock auth until we figure out the user situation?

Reply:
I believe so. Until we determine how we can mimic DOJ's custom group claim, we will need to use mock.

Reply:
We will probably want to validate the functionality in a running environment with both mock and okta auth providers

Reply:
Punt until we can mimic DOJ login.